docs/content/getting-started/setup-openfga.mdx

```diff
@@ -201,16 +213,80 @@ docker run --name openfga --network=openfga -p 3000:3000 -p 8080:8080 -p 8081:80
     --datastore-uri 'root:secret@tcp(mysql:3306)/openfga?parseTime=true'
 ```
 
+</TabItem>
+<TabItem value={"docker-compose-mysql"} label={"Docker Compose"}>
 
-OpenFGA should start and you should see `using 'mysql' storage engine` in the logs.
+Copy the below code block into a local file named: `docker-compose.yaml`
```

Comment:
I vote we cut down on the details of this section and link them to or reference an example instead. The content of this file is already getting larger and lager.

Reply:
I agree that it's long, but in this PR i only updated the MySQL section to be similar to the Postgres section so that they both have the `docker` and `docker compose` tabs. In the future we may consider switching to a `mysql` and `postgres` tabs and put the Docker code there and link to external docker-compose files.